public/js/components/AtomEdit/AtomEdit.js

```diff
@@ -2,6 +2,8 @@ import React, { PropTypes } from 'react';
 import {CTAEditor} from './CustomEditors/CTAEditor';
 import {RecipeEditor} from './CustomEditors/RecipeEditor';
 import {ExplainerEditor} from './CustomEditors/ExplainerEditor';
+import {StoryQuestionsEditor} from './CustomEditors/StoryQuestionsEditor';
```

Comment:
When we're less pushed I think we should move to using the index.js file in the folder trick like ManagedForm